File: 338-counting-bits/338-counting-bits.py
class Solution:
    def countBits(self, n: int) -> List[int]:
        list = []
        for i in range(n+1):
            sum = 0
            num = i
            while num>0:
                sum += num%2
                num = num//2
            list.append(sum)
        return list

File: 338-counting-bits/338-counting-bits.cpp
class Solution {
public:
    vector<int> countBits(int n) {
        vector<int> res;
        for(int i = 0; i<=n; i++){
            int ans = 0;
            int j = i;
            while(j>0){
                j = j &(j -1); 
                ans ++;
            }
            res.push_back(ans);
        }
        return res;
    }
};